lib/workers/qemu.js

```diff
@@ -59,12 +76,25 @@ module.exports = class QEMUWorker {
       '-smp', '4'
     ])
 
-    childKillEventHandler(this.child, 'exit')
-    childKillEventHandler(this.child, 'uncaughtException')
-    childKillEventHandler(this.child, 'unhandledRejection')
+    this.child.stderr.on('data', (data) => {
+      stderr += data
+    })
+
+    this.child.on('exit', (code) => {
+      if (code !== 0) {
```

Comment:
Hm, I'm thinking that this handler might be actually useless. The child can exit after `.on()` returned (i.e. some time in the future), which will cause this exception that can't be handled in any way. We could make `.on()` reject if this is the case, but then we're blocked on waiting for the child, which is not what we want, or we can make `.on()` return an event emitter, which also makes things a lot more complex and doesn't translate to other workers. Therefore I'm thinking that given we can't do anything with this error, then its not worth having it at all.

Reply:
I think we could just print it when the close event gets emitted. The same way we had it before.